### ISSUE TITLE
Easy question about ignore_cooldown and which boolean does what.

### ISSUE THREAD

**Issue by joshoea** [chiyadev/MudaeFarm#157]
Does having ignore_cooldown or kakera_ignore cooldown on false mean "it will NOT attempt to claim characters/kakera on cooldown". Or is it the other way around with having it on true will mean "it will attempt to claim"? I followed the guide with ignore_cooldown on false but it keeps trying to claim characters/kakera.

**Comment by phosphene47** [comment]
I don't see any logical difference either way. Setting it to true means detected characters will always be claimed regardless of cooldown. Characters may still be claimed if the bot doesn't have cooldown information, which is possible in some cases. Otherwise, if the bot knows that you're on cooldown and ignore_cooldown is false, it will not claim.

**Comment by joshoea** [comment]
Ok. However, does setting it to false theoretically mean it will not attempt to claim.

**Comment by phosphene47** [comment]
Yes, if it knows that you're on cooldown and `ignore_cooldown` is false, it will theoretically not claim.

**Comment by joshoea** [comment]
Yep. It still attempts to claim though
